config.py

```diff
@@ -0,0 +1,5 @@
+from os import environ
+
+SECRET_KEY = environ.get('SECRET_KEY')
+SQLALCHEMY_DATABASE_URI = environ.get('DATABASE_URI')
```

Comment:
I haven't tried what the actual result in this case, but `os.environ.get()` will return `None` for a non-existent env var. In the general case, that is likely to produce a cryptic error message when the config parameter is used. It may be better to use `environ[...]` which will produce a `KeyError` for non-existent env vars.

Reply:
Fixed in next commit